packages/amplify-cli/src/extensions/amplify-helpers/remove-env-from-cloud.ts

```diff
@@ -11,32 +12,31 @@ export const removeEnvFromCloud = async (context, envName, deleteS3): Promise<vo
   const { providers } = getProjectConfig();
   const providerPlugins = getProviderPlugins(context);
   const providerPromises: (() => Promise<$TSAny>)[] = [];
-  context.print.info('');
-  context.print.info(`Deleting env: ${envName}.`);
+  printer.blankLine();
+  printer.info(`Deleting env: ${envName}.`);
 
   // Pinpoint attaches an IAM policy to several roles, which blocks CFN from
   // deleting the roles. Work around that by deleting Pinpoint first.
   const categoryPluginInfoList = getAllCategoryPluginInfo(context);
   if (categoryPluginInfoList.notifications) {
-    // eslint-disable-next-line global-require, import/no-dynamic-require, @typescript-eslint/no-var-requires
-    const notificationsModule = require(categoryPluginInfoList.notifications[0].packageLocation);
+    const notificationsModule = await import(categoryPluginInfoList.notifications[0].packageLocation);
     await notificationsModule.deletePinpointAppForEnv(context, envName);
   }
 
-  providers.forEach(providerName => {
-    // eslint-disable-next-line global-require, import/no-dynamic-require, @typescript-eslint/no-var-requires
-    const pluginModule = require(providerPlugins[providerName]);
+  for (const providerName of providers) {
+    const pluginModule = await import(providerPlugins[providerName]);
     providerPromises.push(pluginModule.deleteEnv(context, envName, deleteS3));
-  });
+  }
 
   try {
     await Promise.all(providerPromises);
     await raiseInternalOnlyPostEnvRemoveEvent(context, envName);
   } catch (e) {
-    context.print.info('');
-    context.print.error(`Error occurred while deleting env: ${envName}.`);
-    context.print.info(e.message);
+    printer.blankLine();
+    printer.error(`Error occurred while deleting env: ${envName}.`);
+    printer.info(e.message);
     if (e.code !== 'NotFoundException') {
+      // TODO: not sure why we are ignoring NotFoundException, should let user know at least?
       throw e;
```

Comment:
I guess we should throw an amplifyError error with the trace and resolution

Reply:
updated!